main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,269 @@
+# Purses and Payments
+
+You can store `amount` objects in two kinds of objects:
+- **[`purse`](https://agoric.com/documentation/glossary/#purse)**: Hold
+  an `amount` of same branded digital assets until you withdraw part or
+  all of them into a `payment` for use.
+- **[`payment`](https://agoric.com/documentation/glossary/#purse)**:
+  Hold an `amount` of digital assets to send to another party. Usually either
+  a freshly created `amount` or one taken from a `purse`
+
+Assets in purses and payments do not have to be currency-like, but can
+be anything you can construct a digital rights `amount` object for;
+swords to use in a game, rights to use a particular contract, theater
+tickets, etc.
+
+Each non-empty `purse` and `payment` object contains exactly one
+`amount` object. While you may deposit or withdraw an `amount` to a
+`purse`, it is repsectively added to the `purse` `amount`or split off into a `payment`
+object's `amount`. The same is true for adding to or splitting a
```

Comment:
We cannot remove partial amounts from payments. Thus, nothing is ever split off from a payment in a deposit or withdraw.

Reply:
Guess I wasn't clear here. I was talking about taking value from a purse and putting it into a payment, such that you'd still have just one amount in the purse and one amount in the payment; you'd subtract what you took from the purse from its single thing, and add what to gave to the payment to the payment's single thing, rather than having two separate representations in both the purse and payment. Will rephrase.